src/cli/cmds/pod.toit

```diff
@@ -130,6 +130,90 @@ create-pod-commands -> List:
       --run=:: download it
   cmd.add download-cmd
 
+  tag-cmd := Command "tag"
+      --help="""
+        Change the tags of an uploaded pod.
+        """
+  cmd.add tag-cmd
+
+  tag-add-cmd := Command "add"
+      --help="""
+        Add tags to uploaded pods.
+
+        The pods to change are specified through a pod reference like
```

Comment:
Can this change the tags of multiple pods? A pod reference specifies a single pod, no?

I guess you can pass multiple pod references here, but tags/pods/pod references is in this section are somewhat confusing.

Reply:
I reworded it a bit.